src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,236 @@
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static kitchenpos.TestFixtureFactory.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@Transactional
+@Sql("/truncate.sql")
+@SpringBootTest
+class MenuServiceTest {
+    @Autowired
+    private MenuService menuService;
+
+    @Autowired
+    private ProductDao productDao;
+
+    @Autowired
+    private MenuProductDao menuProductDao;
+
+    @Autowired
+    private MenuGroupDao menuGroupDao;
+
+    @Autowired
+    private MenuDao menuDao;
+
+    @DisplayName("새로운 Menu를 추가할 수 있다.")
+    @ValueSource(ints = {30_000, 32_000})
+    @ParameterizedTest
+    void createMenuTest(int price) {
+        Product product = createProduct("후라이드", BigDecimal.valueOf(16_000));
+        Product savedProduct = productDao.save(product);
+
+        MenuGroup tweChickenMenu = createMenuGroup("두마리메뉴");
+        MenuGroup savedMenuGroup = menuGroupDao.save(tweChickenMenu);
+
+        MenuProduct menuProduct = createMenuProduct(savedProduct.getId(), 2L);
+        Menu menu = createMenu("두마리 후라이드치킨", BigDecimal.valueOf(price), savedMenuGroup.getId(), menuProduct);
+
+        Menu savedMenu = menuService.create(menu);
+
+        assertAll(() -> {
+            assertThat(savedMenu).isNotNull();
+            assertThat(savedMenu.getName()).isEqualTo(menu.getName());
+            assertThat(savedMenu.getPrice()).isEqualByComparingTo(menu.getPrice());
+            assertThat(savedMenu.getMenuGroupId()).isEqualTo(menu.getMenuGroupId());
+            assertThat(savedMenu.getMenuProducts()).hasSize(1);
+        });
+    }
+
+    @DisplayName("예외: 이름이 없는 Menu를 추가")
+    @Test
+    void createMenuWithoutNameTest() {
+        Product product = createProduct("후라이드", BigDecimal.valueOf(16_000));
+        Product savedProduct = productDao.save(product);
+
+        MenuGroup tweChickenMenu = createMenuGroup("두마리메뉴");
+        MenuGroup savedMenuGroup = menuGroupDao.save(tweChickenMenu);
+
+        MenuProduct menuProduct = createMenuProduct(savedProduct.getId(), 2L);
+        Menu menu = createMenu(null, BigDecimal.valueOf(30_000), savedMenuGroup.getId(), menuProduct);
```

Comment:
```suggestion
        Menu menu = createMenu(null, 30_000, savedMenuGroup.getId(), menuProduct);
```

int로 받아 Fixture에서 BigDecimal을 만들어주면 중복을 제거하고 사용성을 높일 수 있을 것 같아요~